reconcile/openshift_rolebindings.py

```diff
@@ -185,3 +196,31 @@ def run(dry_run, thread_pool_size=10, internal=None,
 
     if ri.has_error_registered():
         sys.exit(1)
+
+
+def has_valid_expiration_date(role):
```

Comment:
Type hints, please. In this case, I believe it's 
```suggestion
def has_valid_expiration_date(role: dict[str, Any]) -> bool:
```